codebuild/py311/decrypt_golden_manifest_with_masterkey.yml

```diff
@@ -2,7 +2,7 @@ version: 0.2
 
 env:
   variables:
-    TOXENV: "py311-full_decrypt-mpl"
+    TOXENV: "py311-full_decrypt"
```

Comment:
This removes the MPL from the TestVector execution environment.
Before, the MPL was installed for TestVector execution in this step. But this is a `masterkey` step, so the MPL isn't used.